.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+python: 3.8
+cache: pip
+before_install: pip install -U pre-commit
+script: pre-commit run --all-files
```

Comment:
This'll fail once we start adding actual py files. But let's deal with it in a separate PR.